.github/workflows/CI.yml

```diff
@@ -0,0 +1,49 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  CI:
+    strategy:
+      matrix:
+        backend: ["OPENMP"]
```

Comment:
Somehow `OPENMP` is hard-coded in some test and hence build just `SERIAL` fails. Not sure if worth fixing.

Reply:
I think adding `-DUSE_OMP=0` would fix `SERIAL`

Reply:
Maybe in a later PR.